include/encrypt-decrypt.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include "common.h"
+#include <iostream>
+
+void encrypt(fs::path &input, fs::path &output);
```

Comment:
Functions defined in headers should be declared as extern. You can read about the keyword here <https://en.cppreference.com/w/cpp/language/storage_duration> or here <https://docs.microsoft.com/en-us/cpp/cpp/extern-cpp?view=msvc-160>. It is effectively used to avoid duplicate symbols.

Reply:
This has been added: 3e0d2eb